etc/kayobe/inventory/group_vars/controllers/firewall

```diff
@@ -0,0 +1,448 @@
+    port: 5000
+    proto: tcp
+    destination: 10.65.0.1
+    comment: "hint: used by 'haproxy' in docker container 'haproxy'" 
+  - interface: "lo"
+    port: 6633
+    proto: tcp
+    destination: 127.0.0.1
+    comment: "hint: used by 'neutron-openvsw' in docker container 'neutron_openvswitch_agent'" 
+  - interface: "{{ internal_interface }}"
+    port: 3306
+    proto: tcp
+    destination: 10.65.1.0
+    comment: "hint: used by 'mysqld' in docker container 'mariadb'" 
+  - interface: "breno1.65"
+    port: 3306
+    proto: tcp
+    destination: 10.65.0.1
+    comment: "hint: used by 'haproxy' in docker container 'haproxy'" 
+  - interface: "{{ internal_interface }}"
+    port: 2379
+    proto: tcp
+    destination: 10.65.1.0
+    comment: "hint: used by 'etcd' in docker container 'etcd'" 
+  - interface: "{{ internal_interface }}"
+    port: 11211
+    proto: tcp
+    destination: 10.65.1.0
+    comment: "hint: used by 'memcached' in docker container 'memcached'" 
+  - interface: "breno1.65"
+    port: 11211
+    proto: tcp
+    destination: 10.65.0.1
+    comment: "hint: used by 'haproxy' in docker container 'haproxy'" 
+  - interface: "{{ internal_interface }}"
+    port: 9292
+    proto: tcp
+    destination: 10.65.1.0
+    comment: "hint: used by 'glance-api' in docker container 'glance_api'" 
+  - interface: "{{ internal_interface }}"
+    port: 2380
+    proto: tcp
+    destination: 10.65.1.0
+    comment: "hint: used by 'etcd' in docker container 'etcd'" 
+  - interface: "{{ internal_interface }}"
+    port: 9100
+    proto: tcp
+    destination: 10.65.1.0
+    comment: "hint: used by 'node_exporter' in docker container 'prometheus_node_exporter'" 
+  - interface: "{{ internal_interface }}"
+    port: 8780
+    proto: tcp
+    destination: 10.65.1.0
+    comment: "hint: used by 'httpd' in docker container 'placement_api'" 
+  - interface: "breno1.60"
+    port: 8780
+    proto: tcp
+    destination: 10.60.0.1
+    comment: "hint: used by 'haproxy' in docker container 'haproxy'" 
+  - interface: "breno1.65"
+    port: 8780
+    proto: tcp
+    destination: 10.65.0.1
+    comment: "hint: used by 'haproxy' in docker container 'haproxy'" 
+  - interface: "breno1.60"
+    port: 9292
+    proto: tcp
+    destination: 10.60.0.1
+    comment: "hint: used by 'haproxy' in docker container 'haproxy'" 
+  - interface: "breno1.65"
+    port: 9292
+    proto: tcp
+    destination: 10.65.0.1
+    comment: "hint: used by 'haproxy' in docker container 'haproxy'" 
+  - interface: "{{ internal_interface }}"
+    port: 9101
+    proto: tcp
+    destination: 10.65.1.0
+    comment: "hint: used by 'haproxy_exporte' in docker container 'prometheus_haproxy_exporter'" 
+  - interface: "all"
+    port: 111
+    proto: tcp
+    comment: "hint: used by 'rpcbind'" 
+  - interface: "{{ internal_interface }}"
+    port: 80
+    proto: tcp
+    destination: 10.65.1.0
+    comment: "hint: used by 'httpd' in docker container 'horizon'" 
+  - interface: "all"
```

Comment:
Should probably try to constrain this. It should be just the internal interface.